packages/next-templates/src/app/productpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  ButtonLink,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { IconChevronRight, IconChevronUp } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+import '@/app/styling/css/wmebv.css';
+import '@/app/styling/css/detail-page.css';
+
+import { useEffect } from 'react';
+
+export default function home() {
+  const pageTitle = 'Klacht over de gemeente doorgeven';
+  const websiteLabel = 'gemeente Voorbeeld';
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${pageTitle} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+
+      <PageContent className="voorbeeld-page-content-flex">
+        <Article id="main" className="voorbeeld-article-space ">
+          <BreadcrumbNav label="Kruimelpad">
+            <BreadcrumbNavLink href="/wmebv#">Terug</BreadcrumbNavLink>
+          </BreadcrumbNav>
```

Comment:
Waarom  gebruik je BreadcrumNavLink? hoort dat bij een van de wmebv componenten? De terug Link die ik gebruik heeft ook een pijl ervoor staan :  
```
<Link className="voorbeeld-link-back" href="./">
          <BacklinkIcon />
          Terug
        </Link>
```

Reply:
Hmm, goede vraag. Ik heb deze inderdaad in wmebv gezien en overgenomen. Maar ik vind het misschien wel een betere optie dan alleen link + backlinkicon. Het is een kruimelpad dus in dat geval is het BreadcrumNavLink specifieker dan alleen een link + class denk ik. Bij mij op de pagina staat nu alleen nog "terug" maar het Figma ontwerp laat een langer kruimelpad zien en er zijn op GV ook genoeg lagen voor een kruimelpad (home-contact). Dit had ik toevallig op mijn to-do lijstje staan voor vandaag haha! 

<img width="241" alt="Scherm­afbeelding 2024-03-21 om 10 17 04" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/906aede6-1d0d-471c-a9e0-9f49aa96f030">

Als ik in de Utrecht Storybook documentatie kijk zie ik dezelfde redenering als die ik noemde. Alleen het voorbeeld wat ze daar geven is weer anders dan zowel jouw als mijn optie. 🤷🏻‍♀️
https://nl-design-system.github.io/utrecht/storybook-react/?path=/docs/react-breadcrumb-nav--docs

Als ik in de Figma gemeentevoorbeeld bibliotheek kijk zie ik dit voorbeeld bij groot scherm:
<img width="378" alt="Scherm­afbeelding 2024-03-21 om 10 41 05" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/c76a6ba3-bd65-402b-a606-7e33a2983250">
En de documentatie waarnaar verwezen wordt (https://www.nldesignsystem.nl/breadcrumb/) geeft vooral error's. Op de pagina zelf maar ook als ik naar de links ga kom ik uit op 404. 

Conclusie: ik weet niet wat de juiste optie is en de documentatie maakt me niet veel wijzer.🤷🏻‍♀️🥲